infrastructure/prod.tfvars

```diff
@@ -19,3 +19,4 @@ http_client_max_client_per_route = 40
 #http_client_validate_after_inactivity = 0
 
 definition_cache_latest_version_ttl_sec = 30
+definition_cache_jurisdiction_ttl_sec = 5
```

Comment:
Can we increase both `definition_cache_latest_version_ttl_sec` and `definition_cache_jurisdiction_ttl_sec` to 60 sec on Prod? We must notify services to just wait 1 min after a definition was uploaded. That is an acceptable compromise to get an increase in performance.

Reply:
Done.